azure/parameters/development.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "appServiceDockerImage": {
+      "value": "pezholio/teacher-payment-service"
+    },
+    "databaseName": {
+      "value": "development"
+    },
+    "databaseUsername": {
+      "value": "user"
+    },
+    "databasePassword": {
+      "value": "8dXuq*jvAt-4beU"
```

Comment:
Just to confirm, this is a placeholder and will be reset?

Reply:
Yes. Definitely. All of the secrets in here will be reset when we decide what to do with secrets.